src/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * Copyright (c) 2015 - 2020 Molkobain.
+ *
+ * This file is part of licensed extension.
+ *
+ * Use of this extension is bound by the license you purchased. A license grants you a non-exclusive and non-transferable right to use and incorporate the item in your personal or commercial projects. There are several licenses available (see https://www.molkobain.com/usage-licenses/ for more informations)
+ */
+
+if(!isset($aMolkobainThesaurus)) $aMolkobainThesaurus = [];
+/** @var array $aMolkobainThesaurus Used to centralize from strings and use them in the dict. entries */
+$aMolkobainThesaurus = array_merge($aMolkobainThesaurus, [
+	// Datamodel
+	'Class/Attribute:nb_u' => 'Výška',
+	'Class/Attribute:nb_u+' => 'Výška v jednotkách (U) z %1$s',
+	'Class/Attribute:position_v' => 'Pozice',
+	'Class/Attribute:position_v+/Variant:host' => 'Svislá pozice (U) z %1$s v %2$s (Musí být spodní pozici, nikoli horní)',
+	'Class/Attribute:position_v+/Variant:device' => 'Svislá pozice (U) z %1$s ve skříni (nebo racku pokud je na něj přímo připojen). (Musí být na spodní pozici, nikoli horní)',
+
+	// Fieldsets
+	'Fieldset:baseinfo' => 'Obecné informace',
+	'Fieldset:moreinfo' => 'Více informací',
+	'Fieldset:otherinfo' => 'Další informace',
+	'Fieldset:dates' => 'Datumy',
+]);
+
+// Classes
+Dict::Add('CZ CZ', 'Czech', 'Česky', array(
```

Comment:
```suggestion
Dict::Add('CS CZ', 'Czech', 'Čeština', array(
```

Same as for the previous czech file.